.gitattributes

```diff
@@ -6,6 +6,9 @@ gradlew						text eol=lf
 */src/*/assets/**/*expected*.yml		text eol=lf
 reporter/src/funTest/assets/*-expected-*	text eol=lf
 
+# Use Unix line endings for Freemarker templates for consistency across platforms.
+reporter/src/main/resources/**.ftlh		text eol=lf
```

Comment:
Hmm, doesn't this need to be `reporter/src/main/resources/**/*.ftlh` now?

Reply:
The documentation is a bit unclear on this, we'll see once the Azure Windows test has finished.